Metabase ShinyApp.R

```diff
@@ -345,22 +345,22 @@ server <- function(input, output) {
   output$plot_language <- renderPlotly({plot_language()})
   
   #Workshop format plot and table
-  table_language <- reactive({
+  table_ws_format <- reactive({
```

Comment:
@MargheritaPhilipp the issue was just that you were calling `table_language` for the `ws` and `app version` tables and plots. All sorted now!